tooling-e2e-tests/scheduler_scenario.sh

```
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+ROOT_DIR="${SCRIPT_DIR}/.."
```

Shellcheck (0.11.0)

[warning] ROOT_DIR appears unused. Verify use (or export if used externally).

(SC2034)